snips-nlu-ontology-parsers/src/builtin_entity_parser.rs

```diff
@@ -25,17 +28,20 @@ lazy_static! {
 }
 
 impl BuiltinEntityParser {
-    pub fn new(lang: Language) -> Self {
+    pub fn new(lang: Language, reference_timestamp: i64) -> Self {
```

Comment:
good idea, I'll move this to extract_entities in a moment. I'm wondering on which tier it needs to be optional. It already is optional in the python binding, I can make it optional using Option<i64> in rust, but it will have to be "non-optional" in the ffi. Does that sound ok?